lib/ProofSet.js

```diff
@@ -150,39 +150,34 @@ module.exports = class ProofSet {
   }
 
   /**
-   * Verify Linked Data proof(s) on a document. The proofs to be verified
+   * Verifies Linked Data proof(s) on a document. The proofs to be verified
    * must match the given proof purpose.
    *
    * Important note: This method assumes that the term `proof` in the given
    * document has the same definition as the `https://w3id.org/security/v2`
    * JSON-LD @context.
    *
-   * @param document {object|string} Object with one or more proofs to be
+   * @param {object|string} document - Object with one or more proofs to be
    *   verified, either a string URL (resolved to an object via the given
    *   `documentLoader`) or a plain object (JSON-LD document).
-   * @param options {object} Options hashmap.
-   *
-   * A `suite` option is required:
-   *
-   * @param options.suite {LinkedDataSignature or Array of LinkedDataSignature}
-   *   acceptable signature suite instances for verifying the proof(s).
    *
-   * A `purpose` option is required:
+   * @param {LinkedDataSignature|Array<LinkedDataSignature>} suite -
```

Comment:
```suggestion
   * @param {LinkedDataSignature|LinkedDataSignature[]} suite -
```
Thanks for cleaning up the docs!

This exact usage is described under `Allows one type OR another type (type union)`
https://jsdoc.app/tags-param.html

Reply:
Awesome, I like that notation even better!